docs/reference/deprecated.md

```diff
@@ -12,7 +12,9 @@ Gutenberg's deprecation policy is intended to support backwards-compatibility fo
  - `getCategories` resolvers has been deprecated. Please use `getEntityRecords` resolver instead.
  - `wp.data.select("core").getTerms` has been deprecated. Please use `wp.data.select("core").getEntityRecords` instead.
  - `wp.data.select("core").getCategories` has been deprecated. Please use `wp.data.select("core").getEntityRecords` instead.
+ - `wp.data.select("core").isRequestingCategories` has been deprecated. Please use `wp.data.select("core/data").isResolving` instead.
  - `wp.data.select("core").isRequestingTerms` has been deprecated. Please use `wp.data.select("core").getEntitiesByKind` instead.
```

Comment:
I guess

* `wp.data.select("core").isRequestingTerms` has been deprecated. Please use `wp.data.select("core").getEntitiesByKind`

should be modified to 

* `wp.data.select("core").isRequestingTerms` has been deprecated. Please use `wp.data.select("core").isResolving`